Photons.xcodeproj/xcuserdata/skalra.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This branch needs to be backmerged with `master` to remove `Photons.xcodeproj/xcuserdata/*` files